pandas/core/frame.py

```diff
@@ -2058,18 +2058,24 @@ def to_stata(
         writer.write_file()
 
     @deprecate_kwarg(old_arg_name="fname", new_arg_name="path")
-    def to_feather(self, path) -> None:
+    def to_feather(self, path, **kwargs) -> None:
```

Comment:
any reason not to make these explicit?

Reply:
> any reason not to make these explicit?

It might change with the pyarrow version, needing us to each time update. Passing through kwargs makes this more "future-robust". 

But I could make the ones that there are now explicit. However, that also means that we need to check the pyarrow version to give a nice error message to say which keyword is not yet supported with the older pyarrow versions (which is of course not that difficult)